iped-parsers/iped-parsers-impl/src/test/java/dpf/sp/gpinf/indexer/parsers/LibpffPSTParserTest.java

```diff
@@ -1,44 +1,130 @@
 import org.junit.Test;
 import org.xml.sax.ContentHandler;
 import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
 
-public class LibpffPSTParserTest {
+import dpf.sp.gpinf.indexer.parsers.util.RepoToolDownloader;
 
-    private static InputStream getStream(String name) {
-        return Thread.currentThread().getContextClassLoader().getResourceAsStream(name);
+public class LibpffPSTParserTest extends AbstractPkgTest {
+    private static String testRoot = System.getProperty("user.dir") + "/src/test";
+    private static String osName = System.getProperty("os.name").toLowerCase();
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+        if (osName.startsWith("windows")) {
+            String repoPath = "libyal/libpff/20131028/libpff-20131028.zip";
+            RepoToolDownloader.unzipFromUrl(repoPath, testRoot + "/tmp_tools/");
+            System.setProperty(LibpffPSTParser.TOOL_PATH_PROP, testRoot + "/tmp_tools/pffexport/");
+        }
     }
 
-    // TODO: put assertions here, nothing is being tested...
+    @Override
+    protected void tearDown() throws Exception {
+        if (osName.startsWith("windows")) {
+            File tool_path = new File(System.clearProperty(LibpffPSTParser.TOOL_PATH_PROP));
+            FileUtils.deleteDirectory(tool_path.getParentFile());
+        }
+    }
 
     @Test
-    public void testLibpffPSTParser() throws IOException, SAXException, TikaException {
+    public void testLibpffPSTParserOST() throws IOException, SAXException, TikaException {
+        LibpffPSTParser parser = new LibpffPSTParser();
+        Metadata metadata = new Metadata();
+        ContentHandler handler = new DefaultHandler();
+        parser.setExtractOnlyActive(true);
+        parser.getSupportedTypes(pstContext);
+        metadata.set(TikaCoreProperties.RESOURCE_NAME_KEY, "ost_sample");
+        try (InputStream stream = this.getClass().getResourceAsStream("/test-files/test_sample.ost")) {
+            if (parser.getSupportedTypes(pstContext).isEmpty()) throw new IOException();
+            parser.parse(stream, handler, metadata, pstContext);
+
+            assertTrue(psttracker.messagebody.toString().contains("This is a test email with two attachment files"));
+            assertEquals(4, psttracker.numberofattachments.stream().mapToInt(Integer::parseInt).sum());
+            
+            String attachmentNames = psttracker.attachmentname.toString();
+            String messageDates = psttracker.messagedate.toString();
+            assertTrue(attachmentNames.contains("calibre.docx"));
+            assertTrue(attachmentNames.contains("sample.jpg"));
+            assertTrue(messageDates.contains("2018-08-02T06:15:12"));
+            assertTrue(messageDates.contains("2018-04-24T04:31:17"));
+            assertTrue(psttracker.messagesubject.toString().contains("Microsoft Outlook Test Message"));
 
+        } catch (IOException e) {
```

Comment:
Yeah, that was a poor way of handling the fact that I can't use `Assumptions` like in other tests, because this test case extends AbstractPkgTest which uses jUnit 3.

Reply:
Anyway, it's fixed in 822dbb5

Reply:
Thank you @FelipeFcosta!